R/geom-label.R

```diff
@@ -58,15 +62,17 @@ GeomLabel <- ggproto("GeomLabel", Geom,
   default_aes = aes(
     colour = "black", fill = "white", size = 3.88, angle = 0,
     hjust = 0.5, vjust = 0.5, alpha = NA, family = "", fontface = 1,
-    lineheight = 1.2
+    lineheight = 1.2, linetype = "solid", linewidth = 0.25
   ),
 
   draw_panel = function(self, data, panel_params, coord, parse = FALSE,
                         na.rm = FALSE,
                         label.padding = unit(0.25, "lines"),
                         label.r = unit(0.15, "lines"),
                         label.size = 0.25,
-                        size.unit = "mm") {
+                        size.unit = "mm",
+                        border_colour = NULL,
+                        border_color = border_colour) {
```

Comment:
Would it make sense to add these to the formals of `geom_label()` as well? It might also be best to standardise to UK spelling immediately in `geom_label()`. In addition, I think ggplot2 has settled at using dot.case instead of snake_case for most arguments.

Reply:
added in [3002125](https://github.com/tidyverse/ggplot2/pull/5454/commits/30021259dc7e61ede0494d237e91523e8cf78f0d)

As for dot.case vs. snake_case, it's not consistent, so I'm not sure. `nudge_x` and `nudge_y` are arguments that use snake_case in that same function. And the function name itself is `geom_label`. I'll lean towards underscore unless it's clearly settled. If so, happy to change it :)

Reply:
argument names _should_ use dot.case...

Reply:
Also, if you can think up a name that doesn't have variation between UK/US spelling it would be preferable because we really have gripes with the decision to support both. While we can't backtrack on that decision we can attempt to limit new entrants

Reply:
1. I updated to dot.case. 

2. Yeah, I see how the redundant variables are a headache. I couldn't come up with a good alternative to term for colour/color, so I'm copying what the rest of ggplot does. Open to alternatives!